Rubberduck.Resources/Rubberduck.Resources.csproj

```diff
@@ -21,4 +21,13 @@
       <DependentUpon>$([System.String]::Copy('%(Filename)').Replace('.Designer', '')).resx</DependentUpon>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Icons\Fugue\arrow-180.png" />
+    <None Remove="Icons\Fugue\clock-select.png" />
+    <None Remove="Icons\Fugue\exclamation-pinned.png" />
+    <None Remove="Icons\Fugue\lock-pinned.png" />
+    <None Remove="Icons\Fugue\pinned.png" />
+    <None Remove="Icons\Fugue\plus-circle-pinned.png" />
+    <None Remove="Icons\Fugue\tick-circle-pinned.png" />
+  </ItemGroup>
```

Comment:
Does that really belong here?

Reply:
That appears to be auto-generated code based on the contents of the `Icons\Fugue\` folder (icons that are in the folder but not being used). I'll clean up the icon directory and that should solve this.